src/content/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder.mdx

```diff
@@ -41,6 +41,10 @@ Use the query builder to:
 * Add charts to your dashboards to obtain a complete real-time view of the state of your system.
 * Share your charts with colleagues or users in just two clicks.
 * Get acquainted with querying by using basic mode, then switch to advanced mode to refine your charts using NRQL.
+* Create NRQL alerts from the queries you build and run.
+
+![Query builder](./images/intro_query_builder.png "Query builder")
+**[one.newrelic.com](https://one.newrelic.com) > Query your data**: Run queries of your data to create custom charts, other visualizations, and NRQL alerts.
```

Comment:
<figcaption>**[one.newrelic.com](https://one.newrelic.com) > Query your data**: Run queries of your data to create custom charts, other visualizations, and NRQL alerts.</figcaption>

This would style this like a caption.